Clinical-Genomics/scout
# Move tx overview logic to backend

This PR:
- moves a lot of code preparing the transcripts overview panel from jinja to the backend
- Sorts displayed transcripts by "mane_select" and "mane plus" availability (fix #2862)

**How to test**:
1. Install the branch and notice that tx overview on variant page shows the same info as in master branch
2. Check that MANE transcripts are shown on top of tx overview. To do so, open the POT1 variant in mongoDB compass (_id:"4c7d5c70d955875504db72ef8e1abe77"), open its genes and the second transcript (the one that is shown as second tx in the scout variant page) and add a mane_select_transcript value as illustrated in this pic:
![image](https://user-images.githubusercontent.com/28093618/133384205-71c1d824-8c6b-4fda-a8f1-acf62c2ba07b.png)
3. Switch to master and this transcript is shown second in tx overview
4. In this branch it will be shown on top

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
